0800-appendix-195943-stock.Rmd

```diff
@@ -55,7 +55,7 @@ At the time of reporting, Dogwood Environmental Consulting (Dogwood) were workin
 
 PSCIS stream crossing `r as.character(my_site)` was ranked as a high
 priority for follow up by Canadian Wildlife Federation because of
-significant amounts of habitat modelled as upstream of the crossing. A total of
+significant amounts of habitat modelled as upstream of the crossing. @irvine2018AnalysisPriority ranked the site as a moderate priority for follow up because the stream was dewatered in photos. A total of
```

Comment:
I didnt realize you were on this so quickly but you missed the other crossing anyway so mine stands.